app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -106,7 +106,7 @@ def create_admin_set
       def setup_form
         add_breadcrumb t(:'hyrax.controls.home'), root_path
         add_breadcrumb t(:'hyrax.dashboard.breadcrumbs.admin'), hyrax.dashboard_path
-        add_breadcrumb t(:'hyrax.admin.sidebar.admin_sets'), hyrax.admin_admin_sets_path
+        add_breadcrumb t(:'hyrax.dashboard.my.collections'), hyrax.my_collections_path
```

Comment:
PR #2071 is fixing the breadcrumbs.  I'd prefer not to mix these two PRs.  Can you remove this change from here?

Reply:
Sure.

Reply:
Okay, reverted. It wasn't passing travis ci without some of these changes.